db/migrate/20150105161643_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean
```

Comment:
Couple things:

1) If your `admin` column is a boolean, then it should be either `true` or `false` (because there's no such concept of a `null` boolean). It would be worth adding a default value of either `true` or `false` to this as well (though I'm guessing `false` in this case because you only want to generate admins by choice rather than by default).
2) Instead of `admin` you may want to consider adjusting the column to something like `role`. This way you can have multiple types of users. Maybe one day you'd want three roles: user, mid-level admin, super-admin.